app/src/main/java/app/revanced/integrations/patches/LithoThemeComponentName.java

```diff
@@ -0,0 +1,11 @@
+package app.revanced.integrations.patches;
+
+import static app.revanced.integrations.patches.LithoThemePatch.shortsCommentBox;
+
+public final class LithoThemeComponentName extends Filter {
+    public boolean filter(final String path, final String identifier) {
+        shortsCommentBox = path.contains("comment_composer");
```

Comment:
Probably using a `BlockRule` object here would help with code consistency across extending classes of `Filter`. Consider using it here too.

Reply:
I don't think a blockrule it's needed here. It's simply a check after all, without the need for a return true but only false.

Reply:
> Probably using a `BlockRule` object here would help with code consistency across extending classes of `Filter`. Consider using it here too.

Personal considerations aside: is this change strictly necessary?

Reply:
> I don't think a blockrule it's needed here. It's simply a check after all

A `BlockRule` object is an abstraction for that. I suggest it for code consistency and quality.

Reply:
You would add the same string `comment_composer` as a rule to block, then check for it and return the result to the method.